language/stdlib/modules/LibraVMConfig.move

```diff
@@ -99,12 +124,33 @@ module LibraVMConfig {
         pragma aborts_if_is_partial = true; // TODO: added for a module property. Remove this once the "aborts_if" spec is completely specified.
     }
 
-    public fun set_publishing_option(account: &signer, publishing_option: vector<u8>) {
+    // Add `new_hash` to the list of script hashes that is allowed to be executed by the network.
+    public fun append_script_allow_list(account: &signer, new_hash: vector<u8>) {
```

Comment:
Nit: `add_script? `append` sounds like we are adding another list of scripts to the existing allowlist.